tests/epyccel/test_epyccel_openmp.py

```diff
@@ -243,7 +243,7 @@ def test_omp_matmul_2d_2d(language):
 
 @pytest.mark.parametrize( 'language', [
         pytest.param("c", marks = [
-            pytest.mark.xfail(reason="Numpy Arrays not implemented in C !"),
+            pytest.mark.xfail(reason="for reduction doesn't work in C"),
```

Comment:
@nhamidn confirms that this is fixed in PR #593

Reply:
yes it's fixed